src/listeners/decorators/on.decorator.ts

```diff
@@ -0,0 +1,11 @@
+import { Listener } from 'necord';
+import { NecordLavalinkEvents } from '../listener.interface';
+
+/**
+ * Decorator that marks a method as a listener for Lavalink client.
+ * @param event The event name.
+ * @returns The decorated method.
+ * @url https://necord.org/recipes/lavalink/listeners
+ */
+export const OnLavalink = <K extends keyof E, E = NecordLavalinkEvents>(event: K) =>
```

Comment:
Is it really sends data via client of d.js?

Reply:
What do you mean?

Reply:
@NedcloarBR I mean can I listen it message via `this.client.on` from djs?

Reply:
It is not possible because some of the events are emitted by [LavalinkManager](https://github.com/Tomato6966/lavalink-client/blob/main/src/structures/LavalinkManager.ts) and others by [NodeManager](https://github.com/Tomato6966/lavalink-client/blob/main/src/structures/NodeManager.ts)